### DISCUSSION TITLE
Phillips Hue Centris - Not Supported

### DISCUSSION THREAD

**Discussion by jessevdent** [Koenkk/zigbee2mqtt#12173] [Devices]
Hi,

I just tried to add my new Hue Centris that should be supported: https://www.zigbee2mqtt.io/devices/5061031P7.html
But it shows up as unsupported, maybe that's because of the fixture is recognized as 3 seperate devices:

- 5061030P7_01
- 5061030P7_02
- 5061030P7_03

Can someone please help me?
Thanks in advance

**Comment by Koenkk** [comment]
Added, 

Could you please figure out the color temperature range as described on https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#_3-adding-converter-s-for-your-device ?

Changes will be available in the dev branch in a few hours from now. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Reply by jessevdent** [reply]
Hi, 

Thanks for adding the device!
Sorry it took a while but, these are the color temperature ranges:
2022-04-25 09:14:10Read result of 'lightingColorCtrl': {"colorTempPhysicalMin":153,"colorTempPhysicalMax":500}

**Reply by Koenkk** [reply]
Thanks!
